agents/kathy/src/settings.rs

```diff
@@ -17,18 +17,24 @@ mod test {
     async fn it_builds_settings_from_env() {
         test_utils::run_test_with_env("../../fixtures/env.test", || async move {
             let run_env = dotenv::var("RUN_ENV").unwrap();
-            let agent_home = dotenv::var("AGENT_HOME").unwrap();
+            let agent_home = dotenv::var("AGENT_HOME_NAME").unwrap();
 
             let settings = KathySettings::new().unwrap();
 
             let config = nomad_xyz_configuration::get_builtin(&run_env).unwrap();
-            let secrets = AgentSecrets::from_env(&config.networks).unwrap();
+
+            let remotes = get_remotes_from_env!(agent_home, config);
```

Comment:
Can you add logs to show which network are being loaded?

Reply:
Added log for all remotes loaded